src/App.js

```diff
@@ -1,24 +1,11 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import CounterList from './components/couterList';
```

Comment:
there's a small typo here, would you be up for fixing it? couterList => counterList

Reply:
Thank you @gmzjuliana! We'll review it!